doc/news/9.0.1-vs-9.1.0.h

```diff
@@ -0,0 +1,1976 @@
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+/**
+@page changes_between_9_0_1_and_9_1_0 Changes between Version 9.0.1 and 9.1.0
+
+<p>
+This is the list of changes made between the release of deal.II version
+9.0.1 and that of 9.1.0. All entries are signed with the names of the
+author.
+</p>
+
+
+
+<!-- ----------- INCOMPATIBILITIES ----------------- -->
+
+<a name="incompatible"></a>
+<h3 style="color:red">Incompatibilities</h3>
+
+<p style="color:red">
+Following are a few modifications to the library that unfortunately
+are incompatible with previous versions of the library, but which we
+deem necessary for the future maintainability of the
+library. Unfortunately, some of these changes will require
+modifications to application programs. We apologize for the
+inconvenience this causes.
+</p>
+
+<ol>
+
+ <li>
+  Changed: Tasks of member function
+  hp::FECollection::find_face_dominating_fe_in_subset() are now divided
+  into two functions hp::FECollection::find_common_subspace()
+  and hp::FECollection::find_dominated_fe().
+  <br>
+  (Marc Fehling, 2019/04/08)
+ </li>
+
+ <li>
+  Removed: The IndexSet instantiations for FEValues::get_function_values()
+  and DataOut_DoFData::add_data_vector() have been removed.
+  <br>
+  (Daniel Arndt, 2019/03/20)
+ </li>
+
+ <li>
+  Changed: The "debug" and "release" targets that are created by the
+  DEAL_II_INVOKE_AUTOPILOT() macro no longer automatically rebuild the
+  project.
+  <br>
+  (Matthias Maier, 2019/02/05)
+ </li>
+
+ <li>
+  Changed: VectorSlice has been deprecated in favor of the more general ArrayView
+  class. All public interfaces now take ArrayView arguments instead of VectorSlice
+  arguments. Since VectorSlice now inherits from ArrayView this should be
+  compatible with the overwhelming majority of use cases.
+  <br>
+  (David Wells, 2019/01/21)
+ </li>
+
+ <li>
+  Changed: The class VectorView has been removed. The suggested replacements are
+  to either use an ArrayView, a BlockVector, or to copy the relevant subset into a
+  Vector.
+  <br>
+  (David Wells, 2019/01/11)
+ </li>
+
+ <li>
+  Changed: The particle MPI datatype macro <code>PARTICLE_INDEX_MPI_TYPE</code> is
+  now properly namespaced: its new name is
+  <code>DEAL_II_PARTICLE_INDEX_MPI_TYPE</code>.
+  <br>
+  (David Wells, 2018/11/30)
+ </li>
+
+ <li>
+  Changed: Subscriptor::subscribe cannot be called with rvalue references anymore.
+  In particular, temporary string literals cannot be used as argument. Storing
+  string literals with the same content at the same memory location is
+  implementation defined and cannot be relied upon. On the other hand, the
+  subscriber strings are not compared by their memory address anymore but by their
+  actual content.
+  <br>
+  (Daniel Arndt, 2018/11/12)
```

Comment:
This changelog entry is no longer quite right: we *do* allow these again with #7901.